### DISCUSSION TITLE
Welcome to SimpleWebAuthn Discussions!

### DISCUSSION THREAD

**Discussion by MasterKale** [MasterKale/SimpleWebAuthn#83] [General]
## 👋 Welcome!
  I'm trialing Discussions as a way to encourage conversations about SimpleWebAuthn without creating new Issues for everything.

Please feel free to:
  * Ask questions about the SimpleWebAuthn project, including its individual packages
  * Share ideas for improving the project

Thanks for joining me on this crazy WebAuthn ride 🔐

**Comment by shannonlal** [comment]
@MasterKale Hey I stumbled across this repo as I was recently playing with Webauthn and looking at a couple of other repos.  I commit on a couple of repos and would be willing to help out.  Let me know if there any areas that you would like some help with
